api-reference/10 UI Widgets/dxDataGrid/1 Configuration/customizeExportData.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: dxDataGrid.Options.customizeExportData
 type: function(columns, rows)
-deprecated: Since v20.1, we recommend [ExcelJS-based export](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/) which does not use this option.
+deprecated: Since v20.2, we recommend [ExcelJS-based export](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/) which does not use this option.
```

Comment:
This change is incorrect. You can search for `since v20` and revert all similar changes.